.github/workflows/deploy-all.yml

```
@@ -122,7 +122,11 @@ jobs:
     uses: ./.github/workflows/deploy-aws-lambda.yml
     with:
       environment: ${{ needs.determine-deployment.outputs.environment }}
-    secrets: inherit
+    secrets:
+      ADMIN_EMAIL: ${{ secrets.ADMIN_EMAIL }}
```

actionlint (1.7.12)

[error] secret "ADMIN_EMAIL" is not defined in "./.github/workflows/deploy-aws-lambda.yml" reusable workflow. no secret is defined

(workflow-call)